app/assets/javascripts/Components/Result/pdf_viewer.jsx

```diff
@@ -1,9 +1,14 @@
 import React from "react";
+import {SingleSelectDropDown} from "../../DropDownMenu/SingleSelectDropDown";
 
 export class PDFViewer extends React.PureComponent {
   constructor(props) {
     super(props);
     this.pdfContainer = React.createRef();
+    this.state = {
+      zoom: "page-width",
+      rotationInDegrees: 0,
```

Comment:
Name this `rotation` to be consistent with the `ImageViewer` class. You can leave a comment indicating this is measured in degrees